sphinx/source/docs/installation.rst

```diff
@@ -93,6 +93,20 @@ Or directly from a Bash or Windows command prompt:
 
     python -c "import bokeh.sampledata; bokeh.sampledata.download()"
 
+Finally, the location that the sample data is stored can be configured.
+By default, data is downloaded and stored to a directory ``$HOME/.bokeh/data``.
+(The directory is created if it does not already exist.) Bokeh looks for
+a YAML configuration file at ``$HOME/.bokeh/config``. The YAML key
+``sampledata_dir`` can be set to the absolute path of a directory where
+the data should be stored. For instance adding the following line to the
+config file:
+
+.. code-block:: sh
+
+    sampledata_dir: /tmp/bokeh_data
+
+Will cause the sample data to be stored in ``/tmp/bokeh_data``.
```

Comment:
`will`, I don't think you should use capital letter here... I will modify it and merge this because LGTM.